domain-models-runtime/src/test/java/org/folio/rest/persist/PgUtilIT.java

```diff
@@ -98,7 +120,19 @@ private static void execute(TestContext context, String sql) {
     });
     async.await();
   }
+  private static void executeAndNotify(TestContext context, String sql,Handler<AsyncResult<JsonArray>>  handler ) {
+    Async async = context.async();
+    log.error("execute and notify called");
```

Comment:
Why is this an error? If it isn't, can you log it with a suitable non-error level?

The same applies to the 2 other log.error below.